src/main/java/com/faforever/client/fx/WebViewConfigurer.java

```diff
@@ -2,40 +2,46 @@
 
 import com.faforever.client.preferences.PreferencesService;
 import com.faforever.client.theme.UiService;
+import com.sun.javafx.webkit.Accessor;
 import javafx.concurrent.Worker.State;
+import javafx.event.EventHandler;
 import javafx.scene.input.KeyCode;
+import javafx.scene.input.MouseEvent;
 import javafx.scene.web.WebEngine;
 import javafx.scene.web.WebView;
+import netscape.javascript.JSObject;
 import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Scope;
 import org.springframework.stereotype.Component;
 import org.w3c.dom.Document;
 import org.w3c.dom.Element;
 import org.w3c.dom.NodeList;
-import org.w3c.dom.events.EventListener;
-import org.w3c.dom.events.EventTarget;
-
-import javax.inject.Inject;
 
 @Component
 @Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
 public class WebViewConfigurer {
 
+  /**
+   * This is the member name within the JavaScript code that provides access to the Java callback instance.
+   */
+  private static final String JAVA_REFERENCE_IN_JAVASCRIPT = "java";
```

Comment:
Is this really needed? At some other position you hardcoded "<html></html>". Are you sure this could be subject to change? This seems a bit odd to me, as it is hardcoded anyways as there is JAVA in the name of the variable.

Reply:
Arguable, but my thinking is this. The advantage of
```
((JSObject) engine.executeScript("window")).setMember(JAVA_REFERENCE_IN_JAVASCRIPT, browserCallback);
```
vs
```
((JSObject) engine.executeScript("window")).setMember("java", browserCallback);
```
is that the first one uses a constant with a meaningful name, whose exact content is not relevant to where it's used. At the latest when it's used a second time its use is obvious - I thought it was used more than once but apparently, it's not the case now. Another (small) advantage is that you can add a JavaDoc to the constant, 

To sum it up, it's not needed and there is no big benefit to it with its current single usage. But I wouldn't consider it bad either.